packages/widget-reorder/src/control.tsx

```diff
@@ -47,34 +44,42 @@ export const createControl: CreateControl = (options = {}) => {
         data,
         key: fieldId,
       })
-      this.setState({ data: newOrder })
-      if (modified) onChange(fromJS(newOrder))
-    }
 
-    public handleDragEnd = result => {
-      const { onChange } = this.props
+      this.updateChange(newOrder, modified)
+    }
 
-      if (!result.destination) return
+    private updateChange = (data: object, triggerOnChange = true) => {
+      const { onChange, field } = this.props
+      if (triggerOnChange) {
+        onChange(fromJS(data))
+      }
+      this.setState({ data })
+      const key = generateIdentifierFromField(field)
+      localStorage.setItem(key, JSON.stringify(data))
+    }
 
+    public handleDragEnd = result => {
+      if (!result.destination || result.source.index === result.destination.index) return
```

Comment:
Good call! Dragging item & putting it back to the same spot no longer trigger `onChange`! 👍